sql-odbc/docs/dev/BUILD_INSTRUCTIONS.md

```diff
@@ -2,85 +2,100 @@
 
-#### with Developer Powershell
+### Build
 
-Use `./build_win_<config><bitness>.ps1` to build the driver from a Developer Powershell prompt.
+```
+.\build_win_<config><bitness>.ps1
+```
 
-> A shortcut is installed on your system with Visual Studio (search for **"Developer Powershell for VS 2019"**)
+A Visual Studio 2019 solution will also be generated for building/testing the driver with Visual Studio. (Note: if the build fails, the solution will still be generated.)
 
-> Programs launched with this prompt (ex: VS Code) also have access to the Developer shell.
+Solution file: `<odbc-root>\build\odbc\cmake\global_make_list.sln`.
 
-### Build Output
+### Output
 
 ```
 build
-└-- <Configuration><Bitness>
-  └-- odbc
-    └-- bin
-      └-- <Configuration>
-    └-- build
-    └-- lib
-  └-- aws-sdk
-    └-- build
-    └-- install
+└-- odbc
+  └-- bin
+    └-- <Configuration>
+  └-- cmake
+  └-- lib
+└-- aws-sdk
+  └-- build
+  └-- install
 ```
 
-* Driver DLL: `.\build\<Config><Bitness>\odbc\bin\<Config>\odfesqlodbc.dll`
-* Test binaries folder: `.\build\<Config><Bitness>\odbc\bin\<Config>`
+* Driver DLL: `.\build\odbc\bin\<Config>\odfesqlodbc.dll`
+* Test binaries folder: `.\build\odbc\bin\<Config>\`
 
 ### Packaging
 
-From a Developer Powershell, run:
 ```
-msbuild .\build\Release<Bitness>\odbc\PACKAGE.vcxproj -p:Configuration=Release
+msbuild .\build\odbc\PACKAGE.vcxproj -p:Configuration=Release
 ```
 
-An installer named as `Open Distro for Elasticsearch SQL ODBC Driver-<version>-Windows-<Bitness>-bit.msi` will be generated in the build directory.
+`Open Distro for Elasticsearch SQL ODBC Driver-<version>-Windows-<Bitness>-bit.msi` will be generated in the build directory.
 
+### Testing
+See [run_tests.md](./run_tests.md)
 
 ## Mac
-(TODO: upgrade build scripts & documentation for Mac)
 
-### Dependencies
+### Requirements
 
-Homebrew must be installed to manage packages, to install homebrew see the [homebrew homepage](https://brew.sh/).
-Using homebrew, install the following packages using the command provided:
->brew install [package]
->
->* curl
->* cmake
->* libiodbc
+[Homebrew](https://brew.sh/) is required for installing the following necessary packages.
+```
+brew install curl
+brew install cmake
+brew install libiodbc
+```
 
-### Building the Driver
+### Build
 
-From a Bash shell:
+```
+./build_mac_<config><bitness>.sh
+```
 
-`./build_mac_<config><bitness>.sh`
+### Output
 
-### Output Location on Mac
+```
+build
+└-- odbc
+  └-- bin
+  └-- lib
+└-- cmake-build64
+  └-- <build files>
+```
 
-Compiling on Mac will output the tests to **bin64** and the driver to **lib64**. There are also some additional test infrastructure files which output to the **lib64** directory.
+* Driver library: `./build/odbc/lib/libodfesqlodbc.dylib`
+* Test binaries folder: `./build/odbc/bin/`
 
 ### Packaging
 
-Run below command from the project's build directory.
->cpack .
+```
+cd cmake-build64/
+cpack .
```

Comment:
What I found in ODBC GitHub action is the following 4 commands:

```
$ cd cmake-build64
$ cmake ../src
$ make
$ cpack .
```

Is the cmake and make command also required? Or it's already covered in other section.

Reply:
The cmake & make commands seem to be there to make sure the CMake steps have been run for packaging with GHActions. For active development, these lines shouldn't be needed, but I'll add a note around this block to reflect that these are needed if driver/CMake changes are made

Reply:
Added note to re-run build script before packaging, if driver code/CMake project changes have been made

Reply:
I see. Thanks!